index.html

```diff
@@ -0,0 +1,157 @@
+             </div>
+             <img src="images/voice-search.png" alt="voice-icon" class = "voice-icon">
+        </div>
+
+        <div class="nav-right flex-div" >
+            <img src="images/upload.png" alt="upload-icon">
+            <img src="images/more.png" alt="images-icon">
+            <img src="images/notification.png" alt="notification-icon">
+            <img src="images/Jack.png" alt="menu icon" class = "user-icon">
+        </div>
+    </nav>
+
+    <!-- sidebar -->
+    <div class="sidebar">
+        <div class="shortcut-links">
+            <a href=""><img src="images/home.png" alt="home-icon"><p>Home</p></a>
+            <a href=""><img src="images/explore.png" alt="explore-icon"><p>Explore</p></a>
+            <a href=""><img src="images/subscriprion.png" alt="subscriprion-icon"><p>Subscription</p></a>
+            <a href=""><img src="images/library.png" alt="library-icon"><p>Library</p></a>
+            <a href=""><img src="images/history.png" alt="history-icon"><p>History</p></a>
+            <a href=""><img src="images/playlist.png" alt="playlist-icon"><p>Playlist</p></a>
+            <a href=""><img src="images/messages.png" alt="messages-icon"><p>Messages</p></a>
+            <a href=""><img src="images/show-more.png" alt="show-more-icon"><p>Show-more</p></a>
+            <hr>
+        </div>
+        <div class="subscribed-list">
+            <h3>SUBSCRIBED</h3>
+            <a href=""><img src="images/Jack.png"><p>Jack Nicholson</p></a>
+            <a href=""><img src="images/simon.png"><p>Simon Baker</p></a>
+            <a href=""><img src="images/tom.png"><p>Tom Hardy</p></a>
+            <a href=""><img src="images/megan.png" ><p>Megan Ryan</p></a>
+            <a href=""><img src="images/cameron.png"><p>Cameron Diaz</p></a>
+        </div>
+    </div>
+
+    <!---Main-->
+    <div class="container">
+        <div class="banner">
+            <img src="images/banner.png">
+        </div>
+        <div class="list-container">
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail1.png" class="thumbnail"></a>                
+                <div class = "flex-div">
+                    <img src="images/Jack.png">
+                    <div class="vid-info">
+                        <a href="">Best channel to learn coding</a>
+                        <p>Adorer Tutorials</p>
+                        <p>15k Views &bull; 5 days</p>
+                    </div>
+                </div>
+            </div>
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail2.png" class="thumbnail"></a>                
+                <div class = "flex-div">
+                    <img src="images/Jack.png">
+                    <div class="vid-info">
+                        <a href="">Best channel to learn coding</a>
+                        <p>Adorer Tutorials</p>
+                        <p>15k Views &bull; 5 days</p>
+                    </div>
```

Comment:
I think the use of correct html semantics where appropriate will be better than using a div. (example, section, footer, etc) 💯